### ISSUE TITLE
Cannot run swift test

### ISSUE THREAD

**Issue by Jonathan-Lo-1996** [cui-unige/semantique#1]
With the command `swift test`

I have the following result in my terminal

```
error: terminated(1): '/home/username/Téléchargements/swift-4.0-RELEASE-ubuntu16.04/usr/bin/swift-build-tool' -f /home/username/Documents/semantique/.build/debug.yaml test
```

**Comment by saucisson** [comment]
One warning:

```
/data/.build/checkouts/Funky-5714955810262541138/Sources/Collection.swift:29:18: warning: 'CharacterView' is deprecated: Please use String or Substring directly
extension String.CharacterView : SingletonCollection {
```

And one problem:

```
/data/.build/checkouts/anzen-8993467780056719150/Sources/AnzenAST/Parser.swift:2:8: error: no such module 'os.log'
import os.log
```

**Comment by kyouko-taiga** [comment]
Fixed in d722a359a9cbe5264268cae3377730d568eb8883

The warning comes from [Funky](https://github.com/rxwei/Funky), a dependence of the Anzen's parser. The package hasn't been fully ported to Swift 4+.

**Comment by saucisson** [comment]
You have to delete the `.build` directory, as well as the `Package.resolved` file in order to fix the problem.
